tests/e2e/Provide_Existing_Coverage/infection-coverage/junit.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <testsuites>
-  <testsuite name="Test Suite" tests="2" assertions="2" errors="0" failures="0" skipped="0" time="0.010499">
-    <testsuite name="Namespace_\Test\SourceClassTest" file="tests/SourceClassTest.php" tests="2" assertions="2" errors="0" failures="0" skipped="0" time="0.010499">
-      <testcase name="test_it_adds" class="Namespace_\Test\SourceClassTest" classname="Namespace_.Test.SourceClassTest" file="tests/SourceClassTest.php" line="10" assertions="1" time="0.007975"/>
-      <testcase name="test_it_returns_true" class="Namespace_\Test\SourceClassTest" classname="Namespace_.Test.SourceClassTest" file="tests/SourceClassTest.php" line="16" assertions="1" time="0.002524"/>
+  <testsuite name="Test Suite" tests="2" assertions="2" errors="0" failures="0" skipped="0" time="0.015839">
+    <testsuite name="ProvideExistingCoverage\Test\SourceClassTest" file="/Users/tfidry/Project/Humbug/infection/tests/e2e/Provide_Existing_Coverage/tests/SourceClassTest.php" tests="2" assertions="2" errors="0" failures="0" skipped="0" time="0.015839">
```

Comment:
1. why the path changed from relative to absolute?
2. humbug? :)

Reply:
no idea... I tried to generate it again because the existing one was giving a failure

Reply:
forgive me :P I have way more than I should do under that namespace:

<img width="323" alt="Screen Shot 2020-03-06 at 22 57 20" src="https://user-images.githubusercontent.com/5175937/76125803-e3b6e400-5ffd-11ea-9529-de53bcad9412.png">

Reply:
what kind of failure?

Reply:
I mean, if new code started failing for existing tests, isn't it a marker of introduced bug?

JUnit report may be different for different PHPUnit versions, as well as for different test frameworks, unfortunately

Reply:
It appears so, think it is fixed now